octree/include/pcl/octree/impl/octree_iterator.hpp

```diff
@@ -281,6 +281,65 @@ namespace pcl
 
       return (*this);
     }
+
+    //////////////////////////////////////////////////////////////////////////////////////////////
+    template<typename OctreeT>
+    OctreeFixedDepthIterator<OctreeT>::OctreeFixedDepthIterator (unsigned int depth_arg) :
+        OctreeBreadthFirstIterator<OctreeT> (depth_arg), depth_(depth_arg)
+    {
+    }
+
+    //////////////////////////////////////////////////////////////////////////////////////////////
+    template<typename OctreeT>
+    OctreeFixedDepthIterator<OctreeT>::OctreeFixedDepthIterator (OctreeT* octree_arg, unsigned int depth_arg) :
+        OctreeBreadthFirstIterator<OctreeT> (octree_arg, depth_arg), depth_(depth_arg)
```

Comment:
Same comment as above here. Pass 0 to OctreeBreadthFirstIterator max_depth.

Reply:
Done.

Reply:
Same question as the comment above.
> I would like to call the base constructor with the `fixed_depth_arg` parameter